weasyl/templates/submit/multimedia.html

```diff
@@ -72,7 +72,7 @@ <h3>Description</h3>
 
     <div class="column tagging">
       <h3>Tags</h3>
-      <textarea name="tags" class="input tags-textarea" rows="3"></textarea>
+      <textarea name="tags" class="input tags-textarea" rows="3" required></textarea>
```

Comment:
Likewise, no Python enforcement here. See other comments.

Reply:
I think we enforce this? Or the code seems to be trying to, at least. https://github.com/Weasyl/weasyl/blob/25699ec20c7fc62a8433524733ba2cd2d20e4674/weasyl/submission.py#L97-L98

Reply:
Yep, checked; I get the error for new multimedia submissions on master.